security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,72 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
+
+* Previously, version detection did not work correctly on IBM cloud and Hypershift environments. With this release, Compliance Operator now checks `HostedCluster.Status.Version.History[0]` to report a correct result. (link:https://issues.redhat.com/browse/OCPBUGS-3953[*OCPBUGS-3953*])
+
+* Previously, the instructions for rule `ocp4-etcd-unique-ca` did not work for OCP 4.17 and later. With this update, the instructions and actionable steps are corrected. (link:https://issues.redhat.com/browse/OCPBUGS-42350[*OCPBUGS-42350*])
+
+* When using the Compliance Operator with Cluster Logging Operator (CLO) version 6.0, various rules would fail. This is due to backwards incompatible changes to the CRDs that CLO uses. The Compliance Operator relies on those CRDs to verify logging functionality. The CRDs have been corrected to support the PCI-DSS profiles with CLO. (link:https://issues.redhat.com/browse/OCPBUGS-43229[*OCPBUGS-43229*])
+
+* After installing logging 6.0, users found that the ComplianceCheckResult `ocp4-cis-audit-log-forwarding-enabled` was failing because there was a change in the APIversion of `clusterlogforwarder`. Log collection and forwarding configurations are now specified under the new API, part of the observability.openshift.io API group. (link:https://issues.redhat.com/browse/OCPBUGS-43585[*OCPBUGS-43585*])
+
+* For prior releases of Compliance Operator, the URL in the OCP4 GUI console shows (https://docs.openshift.com/container-platform/4.10/security/compliance_operator/compliance-operator-understanding.html), which is incorrect for versions of OCP after version 4.10. As of OCP version 4.11 and later, the link includes the term "co-concepts", such as: https://docs.openshift.com/container-platform/4.11/security/compliance_operator/co-concepts/compliance-operator-understanding.html. As of this release, the URL shown in the OCP4 GUI has been corrected. (link:http://issues.redhat.com/browse/OCPBUGS-48090[*OCPBUGS-48090*])
+
+//  OCPBUGS-48461 has the "no-doc" label because it should not be documented in the release notes.
+
+//  OCPBUGS-48815 was a documentation bug handled with Paul Bastide consulting, which has already been closed and is not related to the 1.7.0 release of Compliance Operator documentation. 
+
+* For previous releases of Compliance Operator, the scans would generate an error log for the reconcile loop on the operator pod. With this release, the Compliance Operator controller logic is more stable and does not generate the errors for the reconcile loop. (link:https://issues.redhat.com/browse/OCPBUGS-51267[*OCPBUGS-51267*])
+
+* Before this release, periodic CI runs twice a week would fail because the runs are expecting the `partition-for-var-log-audit` rule to be `NOT-APPLICABLE` when the test returns `MANUAL`. This has been fixed with the 1.7.0 release and the weekly CI results now match the expected `MANUAL` status. (link:https://issues.redhat.com/browse/OCPBUGS-52295[*OCPBUGS-52295*])
```

Comment:
I have removed OCPBUGS-52295 from the errata as there is customer impact. It only impacts upstream ci. Could you please remove the release notes for OCPBUGS-52295? Thanks.